frontend/main/src/CLIConfigParsing.cpp

```diff
@@ -460,79 +467,52 @@ using OptionsListEntry = std::tuple<std::string, std::string, std::shared_ptr<cx
     std::function<void(std::string const&, cxxopts::ParseResult const&, megamol::frontend::RuntimeConfig&)>>;
 
 std::vector<OptionsListEntry> cli_options_list =
-    { // config name         option description                                                                 type                                        handler
-        {config_option, "Path to Lua configuration file(s)", cxxopts::value<std::vector<std::string>>(),
-            config_handler},
-        {appdir_option, "Set application directory", cxxopts::value<std::string>(), appdir_handler},
-        {resourcedir_option, "Add resource directory(ies)", cxxopts::value<std::vector<std::string>>(),
-            resourcedir_handler},
```

Comment:
I know this is annoying but it looks like CLIConfigParsing.cpp needs a clang-format pass to avoid formatting conflicts with master.